docs/install/tar.md

```diff
@@ -11,6 +11,9 @@ The tarball installation works on Linux systems and provides a self-contained di
 
 The tarball supports CentOS 7, Amazon Linux 2, Ubuntu 18.04, and most other Linux distributions. If you have your own Java installation and you set `JAVA_HOME` in the terminal, macOS works as well.
 
+Open Distro is no longer being developed and exists only as an archive. Please visit [OpenSearch documentation](https://opensearch.org/docs/latest) to get started with OpenSearch.
+{: .warning}
+
 1. Download the tarball:
```

Comment:
These tarballs don't exist anywhere, so the commands don't work:

```  
% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
```

Do you think we should remove them?

Reply:
I moved the warning in the excerpt above to fill the place where the now unavailable tarballs were. Also removed the curl's for checksum, since they're not available either. I put a simple explanation in their place: (No longer available). Just so someone knows there used to be something there.